src/features/rewards/walletWrapper/index.tsx

```diff
@@ -138,73 +245,79 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
         <StyledWrapper
           id={id}
           compact={compact}
+          notificationShown={notification}
         >
           <StyledHeader>
             {
-              alert && alert.node
-              ? <StyledAlertWrapper>
+              !notification
```

Comment:
Can we do it like we are doing alert, so that we don't need this big if statement

Reply:
Having the notification present isn't as simple as "Is there a notification present? If so, display element, if not, hide it" because I am using the `StyledWrapper` just the same and just changing the inner content for the notification.